### ISSUE TITLE
Cement/mud mixing (reminiscent of GM4's datapack)

### ISSUE THREAD

**Issue by vico93** [enjarai/OmniHopper#17]
I suggest that a feature reminiscent of GM4's [Cement Mixers](https://gm4.co/modules/cement-mixers) datapack to be implemented in OmniHopper with Floppers and Hoppers instead.

In GM4, it allows another datapack-provided block, Liquid Tanks, ["to convert Concrete Powder items into Concrete, at the cost of ⅓ of a Bucket of Water per Concrete Powder."](https://wiki.gm4.co/wiki/Liquid_Tanks/Cement_Mixers) Since Omnihopper implement Floppers that transport fluids, and being a mod implements their blocks in a better way than datapacks could do, i think a sightly different but similar functionality could fit in Omnihopper.

Basically, in my idea, a container (chest) with concrete powder or dirt-like blocks (coarse, normal and rooted) receives water from a flopper positioned above, turning the blocks stored inside it to concrete or mud, consuming water from the flopper in the process.

Sorry if my idea is written roughly, i just saw that datapack while reassembling my modpack and thought it would fit with the thematic of Omnihopper.
